test/uk/gov/hmrc/customs/emailfrontend/viewmodels/EmailVerifiedOrChangedViewModelSpec.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.viewmodels
+
+import org.mockito.Mockito.when
+import org.scalatest.matchers.must.Matchers.mustBe
+import play.api.i18n.Messages
+import play.api.test.Helpers
+import uk.gov.hmrc.customs.emailfrontend.config.AppConfig
+import uk.gov.hmrc.customs.emailfrontend.model.ReferrerName
+import uk.gov.hmrc.customs.emailfrontend.utils.SpecBase
+
+class EmailVerifiedOrChangedViewModelSpec extends SpecBase {
+
+  "EmailVerifiedOrChangedViewModel" should {
+
+    "return correct titleKey for verify journey" in new Setup {
+      val viewModel = EmailVerifiedOrChangedViewModel(
+        email = "test@example.com",
```

Comment:
we could have `test@example.com` as a variable since its repeated in multiple places